packages/katello/postgresql-debversion/postgresql-debversion.spec

```diff
@@ -0,0 +1,65 @@
+Name:     postgresql-debversion
+Version:  1.1.1
+Release:  1%{?dist}
+Summary:  Debian version number type for PostgreSQL
+
+Group:    Applications/System
+License:  GPLv3
+URL:      https://salsa.debian.org/postgresql/postgresql-debversion
+Source0:  https://salsa.debian.org/postgresql/postgresql-debversion/-/archive/v%{version}/postgresql-debversion-v%{version}.tar.gz
+Patch0:   0001-Copy-relevant-code-from-apt-pkg-to-ease-packaging.patch
+
+# BuildRequires: g++
+BuildRequires: postgresql-devel
+# BuildRequires: postgresql-server-devel
```

Comment:
Looks like we can cleanup these commented out build requires.

This is built using postgresql-devel but not the SCL postgresql-devel. Is there anything we need to worry about with differences in versions of Postgresql?

Reply:
you might have missed my comment: https://github.com/theforeman/foreman-packaging/pull/4741#issuecomment-589087128

scl-ificiation is still to come